package.json

```diff
@@ -22,13 +22,13 @@
     "test:ci": "yarn test -- -- --run && yarn run-s test:samples:*",
     "test:samples:vue-query": "cd ./samples/vue-query && yarn && yarn test",
     "test:samples:svelte-query": "cd ./samples/svelte-query && yarn && yarn test",
-    "test:samples:react-query:basic": "cd ./samples/react-query/basic && yarn && yarn test",
-    "test:samples:react-query:custom-client": "cd ./samples/react-query/custom-client && yarn && yarn test",
-    "test:samples:react-query:form-data": "cd ./samples/react-query/form-data && yarn && yarn test",
-    "test:samples:react-query:form-data-mutator": "cd ./samples/react-query/form-data-mutator && yarn && yarn test",
-    "test:samples:react-query:form-url-encoded": "cd ./samples/react-query/form-url-encoded && yarn && yarn test",
-    "test:samples:react-query:form-url-encoded-mutator": "cd ./samples/react-query/form-url-encoded-mutator && yarn && yarn test",
-    "test:samples:react-query:hook-mutator": "cd ./samples/react-query/hook-mutator && yarn && yarn test",
+    "test:samples:react-query:basic": "cd ./samples/react-query/basic && yarn && yarn generate-api && yarn test",
```

Comment:
I think we shouldn't do this, it's done using this script: https://github.com/anymaniax/orval/blob/master/scripts/update-samples.mjs

If you add generate-api into test command - then even if dev forgot to update samples - test on CI will pass, and then next dev will see unrelated changes when updating samples.

Reply:
thanks for pointing it out

I missed `update-samples` script